README.md

```diff
@@ -1 +1,47 @@
 # percona-version-service
+
+Percona version service provides API to share support matrix for PMM products.
+
+For each product we have multiple files in `sources` directory.
+Each file contains supported components list for exact version of product.  
+For example `pmm.2.29.0.pmm-server.json` contains supported components for PMM Server 2.29.0.
+
+
+## How to add new product version into supported versions list.
+
+In this example I'll use PMM Server and PSMDB operator.
+PMM Server uses PSMDB Operator.
+
+### Add new version of PMM
+To be able to add new version of PMM to version service we can copy body of file for latest version of PMM with a new name.  
+For example if we want to add new version of PMM 2.30.0:
+1. we should create new file `pmm.2.30.0.pmm-server.json`. 
+2. Copy body of `pmm.2.29.0.pmm-server.json` to a newly created file.
+3. Update version in `operator` field
+
+### Add new version of PSMDB operator
+To be able to add new version of PSMDB operator to PMM you should update existed `pmm.*.pmm-server.json` file.  
+For example if we want to add PSMDB Operator version `1.12.0` to PMM 2.28.0:
+1. Add new child to `psmdbOperator` field
+2. this child should have the following format
+```json lines
+{
+    "image_path": "[docker image name]",
+    "image_hash": "[docker image hash]",
+    "status": "recommended", // or "available"
+    "critical": false // or true
+}
+```
+3. image path and image hash can be taken from docker hub or by running `docker inspect`
+4. status can be `recommended` or `available`
```

Comment:
I think only one version can be `recommended` and multiple can be `available` so maybe add this info.

Reply:
actually no, I've seen that more than one MongoDB version has a `recommended` status for the PSMDB operator.

Reply:
For operators only one can be `recommended` per major database version, so for PSMDB 5.0 only one version can be recommended and another one can be recommended for PSMDB 4.4.